pkg/ibmcsidriver/server.go

```diff
@@ -137,6 +158,35 @@ func (s *nonBlockingGRPCServer) Setup(endpoint string, ids csi.IdentityServer, c
 	return listener, nil
 }
 
+func (f *realFileOps) Chown(name string, uid, gid int) error {
+	return os.Chown(name, uid, gid)
+}
+
+func (f *realFileOps) Chmod(name string, mode os.FileMode) error {
+	return os.Chmod(name, mode)
+}
+
+func setupSidecar(addr string, fileOps FileOps) error {
+	groupSt := os.Getenv("SIDECAR_GROUP_ID")
```

Comment:
We have defined  `SIDE_CAR_GROUP_ID` in configmap. Typo?

Reply:
Right Ambika. Updated the configMap key to `SIDECAR_GROUP_ID `